cli/config_file.go

```diff
@@ -155,6 +162,91 @@ func SettingsFileReadWrite(state *Settings) *Settings {
 	return settings
 }
 
+// Check prerequisites
+func CheckRequirements() {
+	// getenv
+	envVarValue := os.Getenv("DOCKER_HOST")
+
+	if envVarValue == "" {
+		log.Fatalf("DOCKER_HOST environmental variable is empty or not set")
```

Comment:
I have it empty by default, are you sure it is reason to panic?

Reply:
Yes, since this is how docker's SDK works afaik. You cannot really assume the DOCKER_HOST will be a default unix:///var/run/dokcer.sock, since docker can use contexts (remote tcp/ssh sockets, etc), and also socket can be in other places too.

In my opinion, I'd like to force out a properly set DOCKER_HOST env var, than encounter issues later on. Having the default docker socket which isn't working properly for some reason, like my case, only root can access to that socket, could possibly misguide some users because of the misleading error messages.

Should we assume the DOCKER_HOST should be unix:///var/run/dokcer.sock if empty?

Reply:
i think it would be a nice tradeoff, if we could write a highlighted warning message like: `DOCKER_HOST environmental variable is empty or not set. Using unix:///var/run/dokcer.sock as default`

Reply:
And set default to the user.

Reply:
After playing with DOCKER_HOST, IMHO just drop a warning message as @m8vago said would be okey for me.

Reply:
I rearranged it a bit, now it checks the existence of DOCKER_HOST, if it doesn't it will jump straight to connection phase while printing a yellow ERROR message, but when it's present it will try to parse it, and check validity before trying to connect to the socket.

Reply:
fixed